### PR TITLE
Pull in release notes for v1.2.3 and v1.1.4

### PR DESCRIPTION
**Pull request by azdagron**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.2.3] - 2022-04-12
+
+### Security
+- Updated to Go 1.17.9 to address CVE-2022-24675, CVE-2022-28327, CVE-2022-27536.
```
Comment (evan2645) [inline CHANGELOG.md:6]: nit: usually there is no period and a parenthesized PR number is included at the end

**Review by evan2645 — APPROVED** [review]
lgtm, one non-blocking nit

**Comment by azdagron** [comment]
@evan2645 I've removed the trailing period. Please take a look at your leisure :)

**Review by evan2645 — APPROVED** [review]
Thank you @azdagron!

**Pull request merged**